Storage-models.md

```diff
@@ -27,36 +27,21 @@
 ## Introduction
 
 You might be used to programs holding data in variables, but for an application like Oppia, we need some way to store that data such that it survives the application shutting down. We don't want to lose all our data just because we release a new version of Oppia! To store data persistently, you might think about storing it in a file like this:
-
-```text
- User        Application               File system
- ----        -----------               -----------
-  |  GET /prefs   |                         |
-  | ------------> |                         |
-  |               |   f = open('f.json')    |
-  |               |   data = json.load(f)   |
-  |               | ----------------------> |
-  |               |                         |
-  |               |          data           |
-  |               | <---------------------- |
-  | data['prefs'] |                         |
-  | <------------ |                         |
+```mermaid
+sequenceDiagram
+    User ->>Application: GET /prefs
+    Application ->> File System: f = open('f.json'), data= json.load(f)
+    File System -->>Application: data
+    Application -->>User: data['prefs']
```

Comment:
Can we remove the bottom boxes? It is the same as the lines above.

Reply:
I can't remove the bottom boxes. That's a feature in mermaid.
You can check it [here](https://mermaid.js.org/syntax/sequenceDiagram.html)